src/plugins/dashboard/public/application/components/dashboard_listing/__snapshots__/dashboard_listing.test.tsx.snap

```diff
@@ -5789,7 +5785,6 @@ exports[`dashboard listing renders warning when listingLimit is exceeded 1`] = `
                 "observers": Array [],
                 "thrownError": null,
               },
-              "renderWorkspaceMenu": [MockFunction],
               "workspaceEnabled$": BehaviorSubject {
```

Comment:
Need update

Reply:
This PR doesn't include change of `workspaceEnabled$`.